docs/core-concepts/testing.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: doc
-title: Testing
+title: Testing (Everest)
 ---
 
 # Testing
 
-In the case you don't have a charger that supports OCPP 2.0.1 to experiment with, we can recommend using the Linux Foundation Energy project EVerest.
-[See here](https://github.com/EVerest) for the repository.
-They have built an open source version of charger firmware and also allow for using it as a simulator.
-They support OCPP 2.0.1 which makes it a great testing opportunity with CitrineOS.
-For the long route of setting up EVerst you can follow their documentation and build the project yourself.[See here for Docs](https://everest.github.io/latest/general/03_quick_start_guide.html)
+In the case you don't have a charger that supports OCPP 2.0.1 to experiment with, we can recommend using the Linux 
+Foundation Energy project EVerest. [See here](https://github.com/EVerest) for the repository.  They have built an open source version of
+charger firmware and also allow for using it as a simulator. They support OCPP 2.0.1 which makes it a great testing 
+opportunity with CitrineOS. For the long route of setting up EVerst you can follow their documentation and build 
+the project yourself.[See here for Docs](https://everest.github.io/latest/general/03_quick_start_guide.html)
 
-As a short-cut you can also use their demo repository that hosts a Docker packaged EVerest image. [See here for Github Repo](https://github.com/EVerest/everest-demo)
+# Running Everest
+In order to alleviate some of the complexities that may arise when starting Everest, we have created
+some helpful commands that should help in getting the Everest charger simulator running locally and targeting
+CitrineOS.
 
-If you want to run EVerest on the side while developing and making changes, you can follow the steps below.
+You will notice in `/Server/everest` directory the new files created to support running Everest within Docker.
```

Comment:
I would avoid this wording where you say "new files" as these files may be new now, but will not be so new as time goes on.

I would just word it as though the files have always been there.

Reply:
adjusted